helm_deploy/values-dev.yaml

```diff
@@ -5,7 +5,10 @@ generic-service:
   replicaCount: 2
 
   ingress:
+    enabled: true
```

Comment:
suggest you put these values in the `values.yaml` in `helm_deploy/manage-recalls-ui/values.yaml` as they are common to all environments.